src/test/java/org/killbill/billing/plugin/adyen/recurring/AdyenRecurringClient.java

```diff
@@ -0,0 +1,46 @@
+package org.killbill.billing.plugin.adyen.recurring;
```

Comment:
I see. IDEA folds the Copyright Headers, and so I wasn't aware that they're basically everywhere … :grinning: